gluten-ut/spark32/src/test/scala/org/apache/spark/sql/catalyst/expressions/GlutenMathExpressionsSuite.scala

```diff
@@ -17,11 +17,107 @@
 package org.apache.spark.sql.catalyst.expressions
 
 import org.apache.spark.sql.GlutenTestsTrait
+import org.apache.spark.sql.catalyst.dsl.expressions._
+import org.apache.spark.sql.types._
 
 class GlutenMathExpressionsSuite extends MathExpressionsSuite with GlutenTestsTrait {
 
   override def testNameBlackList: Seq[String] = super.testNameBlackList ++ Seq(
     "bin"
   )
 
+  test("Gluten - round/bround") {
```

Comment:
Gluten -> GlutenTestConstants.GLUTEN_TEST?

Reply:
fixed it.